deeposlandia/train.py

```diff
@@ -123,6 +123,96 @@ def add_training_arguments(parser):
                         help=("Number of validation images"))
     return parser
 
+def iou(actual, predicted):
+    """Compute Intersection over Union statistic (i.e. Jaccard Index)
+
+    See https://en.wikipedia.org/wiki/Jaccard_index
+
+    Parameters
+    ----------
+    actual : list
+        Ground-truth labels
+    predicted : list
+        Predicted labels
+
+    Returns
+    -------
+    float
+        Intersection over Union value
+    """
+    actual = backend.flatten(actual)
+    predicted = backend.flatten(predicted)
+    intersection = backend.sum(actual * predicted)
+    union = backend.sum(actual) + backend.sum(predicted) - intersection
+    return 1. * intersection / union
+
+def iou_loss(actual, predicted):
```

Comment:
I think if we have more loss/metrics functions, we will move them into a dedicated Python module.